Misc/NEWS.d/next/Documentation/2019-01-21-14-30-59.bpo-35803.yae6Lq.rst

```diff
@@ -0,0 +1,3 @@
+Document and test that ``tempfile`` functions may accept a
+:term:`path-like object` for the ``dir`` argument.  Patch by Anthony Sottile.
+is a :term:`path-like object`.  Patch by Anthony Sottile.
```

Comment:
The suggestion went awry here; this last line needs to go away.

Reply:
oops! fixed